### PR TITLE
Adds 7.2.4-rc1

### PR DESCRIPTION
**Pull request by roshkhatri**
Adds 7.2.4-rc1 and updates docker hub descriptions with new tags

### CODE REVIEW

File: .github/workflows/update-dockerhub-docs.yml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 name: Update Dockerhub description
 
 on:
-    push:
-    workflow_dispatch:
+  workflow_run:
+    workflows: [GitHub CI]
```
Comment (roshkhatri) [inline .github/workflows/update-dockerhub-docs.yml:5]: This will only run once we push the images to dockerhub

File: versions.sh [hunk 2]
```diff
@@ -137,9 +137,6 @@ for version in "${versions[@]}"; do
 						| (
 							startswith(env.rcVersion + ".")
 							or startswith(env.rcVersion + "-")
-						) and (
-							index("-")
```
Comment (roshkhatri) [inline versions.sh:141]: This script did not allow `-rc` versions to get published

File: versions.json [hunk 3]
```diff
@@ -1,4 +1,60 @@
...
+    "version": "7.2.4-rc1",
+    "url": "https://github.com/valkey-io/valkey/archive/refs/tags/7.2.4-rc1.tar.gz",
+    "sha256": "cd0a74184f2e6addac458f7b4f7549521e490a4ef51db0eb1dccd1a520de4cb6",
+    "debian": {
+      "version": "bookworm"
+    },
+    "alpine": {
+      "version": "3.19"
+    },
+    "gosu": {
```
Comment (hpatro) [inline versions.json:12]: What's this for?

File: dockerhub-description.md [hunk 4]
```diff
@@ -7,8 +7,10 @@
 	Please open an Issue stating your question on [the Valkey Community](https://github.com/valkey-io/valkey-container/issues).
 
 # Supported tags and respective `Dockerfile` links
-- [`unstable`](https://github.com/valkey-io/valkey-container/blob/master/unstable/debian/Dockerfile)
-- [`unstable-alpine`](https://github.com/valkey-io/valkey-container/blob/master/unstable/alpine/Dockerfile)
+- [`unstable`, `unstable-bookworm`](https://github.com/valkey-io/valkey-container/blob/master/unstable/debian/Dockerfile)
```
Comment (hpatro) [inline dockerhub-description.md:10]: Any reason why debian doesn't need to be added as suffix ?

**Review by hpatro — APPROVED** [review]
For each os type I presume the Dockerfile and docker-entrypoint.sh file is auto generated.

File: versions.json (hunk 3 above)
Comment (roshkhatri) [reply versions.json:12]: Its golang super user https://github.com/tianon/gosu
It is used to step down from root during container startup, specially for entrypoint.

**Comment by roshkhatri** [comment]
>For each os type I presume the Dockerfile and docker-entrypoint.sh file is auto generated.

it is done by `apply-templates.sh `using `Docker.template`

**Pull request merged**